mkdocs.yml

```diff
@@ -103,7 +104,6 @@ nav:
         - Google Cloud Storage source: quix-connectors/quix-streams/sources/coming-soon/GoogleCloudStorage-source.md
         - Google Sheets source: quix-connectors/quix-streams/sources/coming-soon/GoogleSheets-source.md
         - Keen source: quix-connectors/quix-streams/sources/coming-soon/Keen-source.md
-        - Kinesis source: quix-connectors/quix-streams/sources/coming-soon/Kinesis-source.md
```

Comment:
@daniil-quix One thing to remember: remove the `quix-connectors/quix-streams/sources/coming-soon/Kinesis-source.md` file.